site/source/api/datumaro.rst

```diff
@@ -0,0 +1,94 @@
+Datumaro
+########
+
+For external use, the recommended way to call Datumaro API is:
+
+.. code-block:: python
+
+    import datumaro as dm
+
+Once you can use the components:
+
+* **dm.annotation** :mod:`datumaro.components.annotation`
+    * **dm.NO_GROUP**
```

Comment:
Please add links to these symbols too.

I think, it would be more convenient to have the whole line as a link instead of just a right half.